kafka-inmemory/resources/openshift-template.yaml

```diff
@@ -1,3 +1,107 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: barnabas-cluster-controller
+  labels:
+    app: barnabas
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
From OpenShift documentation it seems that apiVersion is just v1 and ... order as above :-)

Reply:
@ppatierno I will change the order. But my OpenShift 3.7 seems to support only `rbac.authorization.k8s.io/v1beta1` (you can check it on https://127.0.0.1:8443/apis).